### PR TITLE
LPS-157168 baseline

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-157168
[Aggregation Mock Link](https://www.figma.com/file/LJTlp63zShNRA5hiN5ZdJz/34-LPS-149049-autofill-a-field-with-formulas-considering-values?node-id=2059%3A128080)
[Aggregation Functions and Operators Link](https://liferay.atlassian.net/wiki/spaces/DXPL/pages/2053505487/Aggregation+-+Functions+and+Operators)

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java [hunk 1]
```diff
@@ -629,17 +630,37 @@ private void _addOrUpdateObjectFieldSettings(
 					newObjectFieldSetting.getName());
 
 			if (oldObjectFieldSetting == null) {
-				_objectFieldSettingLocalService.addObjectFieldSetting(
-					objectField.getUserId(), objectField.getObjectFieldId(),
-					newObjectFieldSetting.getName(),
-					newObjectFieldSetting.getValue());
+				if (GetterUtil.getBoolean(
+						PropsUtil.get("feature.flag.LPS-156704")) &&
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java:643]: @marcelabc can you suppress this FF from here? They should be only in the headless layer.

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by gabrielwas** [comment]
LGTM :D

**Review by gabrielwas — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:stop

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:forward

**Comment by danvb112** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:stop

**Pull request closed without merging**